### PR TITLE
Fix multirate splitting test

### PR DESCRIPTION
**Pull request by jkozdon**
fixes how to interpret the multirate test from RobertsSarsharSandu2018arxiv

### CODE REVIEW

**Comment by jkozdon** [comment]
@lcw and @mwarusz: I think that I interpreted the fast and slow tendencies from RobertsSarsharSandu2018arxiv in correctly. When you get a chance can you look and see how you would have interpreted things. If you agree with the new interpretation feel free to merge.

File: test/ODESolvers/runtests.jl [hunk 1]
```diff
@@ -386,21 +386,19 @@ let
     @inbounds begin
       increment || (dQ .= 0)
       yf = Q[1]
-      ys = Q[2]
-      gf = (-3 + yf^2 - cos(ω * t)) / 2yf
-      gs = (-2 + ys^2 - cos(    t)) / 2ys
-      dQ[1] += Ω[1,1] * gf + Ω[1, 2] * gs - ω * sin(ω * t) / 2yf
+      g = @SVector [(-3 + yf^2 - cos(ω * t)) / 2yf, 0]
+      dQ .+= Ω * g
+      dQ[1] -= ω * sin(ω * t) / 2yf
```
Comment (lcw) [inline test/ODESolvers/runtests.jl]: I guess I am missing something, but I think what you previously had was correct.

File: test/ODESolvers/runtests.jl [hunk 2]
```diff
@@ -386,21 +386,19 @@ let
...
     @inbounds begin
       increment || (dQ .= 0)
-      yf = Q[1]
       ys = Q[2]
-      gf = (-3 + yf^2 - cos(ω * t)) / 2yf
-      gs = (-2 + ys^2 - cos(    t)) / 2ys
-      dQ[2] += Ω[2,1] * gf + Ω[2, 2] * gs - sin(t) / 2ys
+      g = @SVector [0, (-2 + ys^2 - cos(    t)) / 2ys]
+      dQ .+= Ω * g
+      dQ[2] -= sin(t) / 2ys
```
Comment (lcw) [inline test/ODESolvers/runtests.jl]: Again, I guess I am missing something, but I think what you previously had was correct.

Comment (jkozdon) [reply test/ODESolvers/runtests.jl]: ok

**Pull request closed without merging**